fastlane/lib/fastlane/actions/gradle.rb

```diff
@@ -94,6 +94,20 @@ def self.run(params)
         return result
       end
 
+      def self.step_text(params)
```

Comment:
Is there any way we could the task name (and others) without changing the method signatures?

Reply:
Not without a rearchitecture 😬 Actions and their methods are execute as class methods and not as instance methods so there is no shared state that `step_text` can get 😢

Reply:
Understood.